src/CustomerBundle/Admin/AddressAdmin.php

```diff
@@ -92,9 +110,16 @@ public function configureListFields(ListMapper $list)
      */
     protected function configureDatagridFilters(DatagridMapper $filter)
     {
+        // NEXT_MAJOR: Keep FQCN when bumping Symfony requirement to 2.8+.
+        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
+            $addressTypeType = 'Sonata\CustomerBundle\Form\Type\AddressTypeType';
+        } else {
+            $addressTypeType = 'sonata_customer_address_types';
```

Comment:
There are two address types in this bundle: `Sonata\CustomerBundle\Form\Type\AddressTypeType` and`Sonata\CustomerBundle\Form\Type\AddressType`
Thats why I named this variable `addressTypeType`
Maybe name it `addressTypesType` ?